server/src/integrations/currency/transactions.ts

```diff
@@ -79,18 +79,15 @@ export async function UpdateTransaction(transactionID: number, deltaCents: numbe
     // get last update
     const lastCharges = await TransactionRepo.getLastChargesForTransactionById(transactionID);
     if (lastCharges != undefined) {
-        console.log("Have history", lastCharges);
```

Comment:
this one was left in by mistake but the console.errors are useful for debugging transactions gone wrong. Could you revert those 2